omc3/plotting/plot_tfs.py

```diff
@@ -531,8 +536,22 @@ def _create_plots(fig_collection, opt):
         output_plot(fig_container)
 
     if opt.show:
+        if len(fig_collection) > 1 and is_qtpy_installed():
+            window = SimpleTabWindow("Tfs Plots")
+            for fig_container in fig_collection.figs.values():
+                tab = PlotWidget(fig_container.fig, title=fig_container.title)
+                window.add_tab(tab)
+            window.show()
+            return
+        
+        if len(fig_collection) > rcParams['figure.max_open_warning']:
+            log_no_qtpy_many_windows()
+
+        for fig_container in fig_collection.figs.values():
```

Comment:
Is this needed if we `plt.show()` afterwards? As far as I understand this, `plt.show()` would spawn a window for each figure anyway? I guess what I don't understand here is the added value